.github/workflows/chart_update_on_merge.yml

```diff
@@ -23,9 +23,10 @@ jobs:
         repository: broadinstitute/cromwell
         token: ${{ secrets.BROADBOT_GITHUB_TOKEN }} # Has to be set at checkout AND later when pushing to work
         path: cromwell
-    - uses: olafurpg/setup-scala@v10
+    - uses: actions/setup-java@v4
       with:
-        java-version: adopt@1.11
```

Comment:
This action [is unmaintained](https://github.com/olafurpg/setup-scala/releases/tag/v14) and does not support our Java 17 distribution.